guide/interactions/buttons.md

```diff
@@ -0,0 +1,321 @@
+# Buttons
+
+With the components API, you can create interactive message components. In this page, we'll be focusing on how to send, receive, and respond to buttons using discord.js!
+
+::: danger
+This page is a follow-up to the [CommandInteraction preview guide pages](https://deploy-preview-638--discordjs-guide.netlify.app/interactions/registering-slash-commands.html). Please carefully read those first so that you can understand the methods used in this section.
```

Comment:
update to not link a deploy preview, please, will try to merge v13 PR's into v13-prep quite quickly, so we can have all of v13 in a WIP PR and have a single deploy preview from there

Reply:
preview linking was just a bridge solution proposed by Sanc to temporarily prevent people running into 404s, will adjust.